Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.cxx

```diff
@@ -102,6 +102,9 @@ vtkMRMLMarkupsDisplayNode::vtkMRMLMarkupsDisplayNode()
   this->LineColorFadingSaturation = 1.;
   this->LineColorFadingHueOffset = 0.;
 
+  this->OccludedVisibility = false;
+  this->OccludedOpacity = 0.75;
```

Comment:
I find this default a bit too high - it is barely distinguishable from full-opacity. What do you think about setting in to 0.3 or 0.5?

0.75:

![image](https://user-images.githubusercontent.com/307929/91469908-0665df00-e862-11ea-94a2-762de7ec6980.png)

0.5:

![image](https://user-images.githubusercontent.com/307929/91469929-0c5bc000-e862-11ea-9a7b-592666baa144.png)

0.3:

![image](https://user-images.githubusercontent.com/307929/91469941-1251a100-e862-11ea-84f1-9b7d0838bf0e.png)

Reply:
Right. I actually meant to set it to 0,25. It was 0.75 from when I had changed the variable to be "Fade".
I think 0.3 makes sense.